NitroxModel-Subnautica/DataStructures/GameLogic/ExosuitModel.cs

```diff
@@ -12,22 +11,28 @@ namespace NitroxModel_Subnautica.DataStructures.GameLogic
     [ProtoContract]
     public class ExosuitModel : VehicleModel
     {
-        [ProtoMember(10)]
+        [ProtoMember(1)]
```

Comment:
I think there is a 10 because I counts from VehicleModel which ends with 9. But don't know exactly if this is important in ProtoBuf.

Reply:
Indeed it's not really important with Protobuf, we can start to count from the scratch even if it's a derivated type with protomembers in the base class